spec/plugin_spec_helper.rb

```diff
@@ -21,19 +21,19 @@ module OpenProject
   module GlobalRoles
     module PluginSpecHelper
       def mobile_tan_plugin_loaded?
-        plugin_loaded?("openproject_mobile_otp")
+        plugin_loaded?('openproject_mobile_otp')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.